packages/core/src/commands/view/ComponentDrag.ts

```diff
@@ -127,7 +125,7 @@ export default {
   },
 
   getGuidesStatic() {
-    let result: Guide[] = [];
+    let result: ComponentGuide[] = [];
```

Comment:
I had to change the name because when generating the build, I found out there's already a Guide type, now it's unique

Reply:
Yeah, correct, all TS types have to be unique as everything is exported in a single file.

I'm pushing a rename to `ComponentDragGuide` as it gives a bit more context.